test/e2e/configs.go

```diff
@@ -22,6 +22,7 @@ const (
 	rules       testType = "rules"
 	logs        testType = "logs"
 	traces      testType = "traces"
+	tracetempls testType = "tracetempls"
```

Comment:
```suggestion
	tracesTemplate testType = "tracesTemplate"
```

I think something like this would be easier to read